docs/frontend_utilisation_guide.md

```diff
@@ -24,48 +24,91 @@ The Pixel Streaming Frontend UI contains all the functionality for UI components
 
 ---
 
-## Getting Started
-### Installing the Scalable Pixel Streaming Frontend   
+## Scalable Pixel Streaming Frontend Pieces
+### The Library
+The Library is the part of the Scalable Pixel Streaming Frontend that consumes the Epic Games Pixel Streaming Frontend and UI Frontend. It includes all of the custom signalling server logic that Scalable Pixel Streaming signalling servers require to work. The Library can be either obtained through [Github](https://github.com/ScalablePixelStreaming/Frontend) or [NPM](https://www.npmjs.com/package/@tensorworks/libspsfrontend). To make use of the Library it must be initialised via HTML and Javascript. The Example below is but one example of initialize the Library.  
+
+### The Typescript Example
+The Typescript Example is a simple HTML, CSS and TypeScript implementation of what a user could create to initialize the Scalable Pixel Streaming Frontend Library. Its role is to instantiate the library's components and help start the Scalable Pixel Streaming connection to the signalling server. Please note  how a user chooses to initialize Library is open to interpretation and the users requirements and is not limited to the Typescript Example.
```

Comment:
From the following sentence at the end of this paragraph:

> Please note  how a user chooses to initialize Library is open to interpretation and the users requirements and is not limited to the Typescript Example.

I think we can drop this sentence as the addition of the sentence in the above *Library* section explains this.

Reply:
resolved in 5d619d6a05dd5c046f3a85d110099bde8723fd61